doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3532,6 +3606,13 @@ NCSDK-30284: The :ref:`radio_test` sample only transmits the first burst of data
 Other samples
 =============
 
+.. rst-class:: v3-2-0
+
+NCSDK-34929: The :ref:`nrf_compression_mcuboot_compressed_update` sample does work correctly on nRF54H20
+  After updating a compressed image for app/radio slot, MCUBoot fails to verify the start address of the binary.
+
+  **Affected platforms:** nRF54H20
+
```

Comment:
Line 3684: This PSK can be obtain from the carrier. -> This PSK can be obtained from the carrier.